docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
     ports:
       - "5555:5555"
       - "5554:5554"
-      - "2112:2112"
+      - "${FULCIO_METRICS_PORT:-2112}:2112"
```

Comment:
i would do 
```suggestion
      - "${FULCIO_METRICS_PORT:-2113}:2112"
```

Reply:
LGTM. You can also use set the `metrics-port` flag to set a different metrics port for the service.

Reply:
@cpanato I didn't want to change the default since it could be considered a breaking change. Do you think having a better default is worth that breaking change?

@haydentherapper I wanted to run this in docker-compose without editing `docker-compose.yml`, so it doesn't really matter to me what port metrics is running on in the container, only what external port is exposed.